app/Services/AmmortizationService.php

```diff
@@ -135,13 +136,26 @@ public function getAllowance($salary){
         return (float)$salary / 4;
     }
 
-    public function confirmMonth($balances, $repayment){
+    public function confirmMonth($balances, $repayment, $customer_type, $index){
         $affirm = 0;
-        foreach($balances as $balance){
+        if($index !== 0 && $customer_type == 'new_customer'){
+            foreach($balances as $balance){
+                if($balance >= $repayment){
+                    $affirm++;
+                }
+            }
+
+            return $affirm >= 2;
+        }
+        else{
+            foreach($balances as $balance){
```

Comment:
why are we repeating this? the if condition should only be on `$affirm >=`
`return ( $index !== 0 && $customer_type == 'new_customer') ? $affirm >= 2 : $affirm >= 1`